examples/tutorial_server_historicaldata.c

```diff
@@ -0,0 +1,121 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+#include "open62541.h"
```

Comment:
due to recent changes to how the examples are handled you can't use the combined header here.
needs to become something like this

-#include "open62541.h"
+#include <ua_server.h>
+#include <ua_config_default.h>
+#include <ua_log_stdout.h>
+#include <ua_plugin_historydatabase.h>
+#include <historydata/ua_plugin_history_data_gathering.h>
+#include <historydata/ua_historydatagathering_default.h>
+#include <historydata/ua_historydatabase_default.h>
+#include <historydata/ua_historydatabackend_memory.h>

Reply:
Done

Reply:
Not completely correct ;)

```
#include <historydata/ua_plugin_history_data_gathering.h>
#include <historydata/ua_historydatagathering_default.h>
#include <historydata/ua_historydatabase_default.h>
#include <historydata/ua_historydatabackend_memory.h>
```

Should not use the historydata folder, since it will not work with the installed target. make install just copies all the header files into one folder and then this code will fail.

You need to change it to 

```
#include <ua_plugin_history_data_gathering.h>
#include <ua_historydatagathering_default.h>
#include <ua_historydatabase_default.h>
#include <ua_historydatabackend_memory.h>
```

Reply:
Is this really true?

It will not build if I change that.

Maybe, this is true for examples that will be build out of library source, but not if build inside library source.

Reply:
I tried if the ci is successful building it without folder names in the includes.
It went wrong.
I reverted that commit.
So, path need to be there.